From: Ember <ember@users.noreply.github.com>
Subject: [PATCH 7/74] dlls-d3dx9_25: Ember changes

Base: CrossOver 26.3.0 FOSS sources/wine
License: same as the upstream file (LGPL-2.1-or-later for Wine/DXMT; see LICENSES/)
---
diff --git a/dlls/d3dx9_25/d3dx9_25.spec b/dlls/d3dx9_25/d3dx9_25.spec
index 6fa19bf..d6dba64 100644
--- a/dlls/d3dx9_25/d3dx9_25.spec
+++ b/dlls/d3dx9_25/d3dx9_25.spec
@@ -163,7 +163,7 @@
 @ stub D3DXGetTargetDescByVersion
 @ stdcall D3DXGetVertexShaderProfile(ptr)
 @ stdcall D3DXIntersect(ptr ptr ptr ptr ptr ptr ptr ptr ptr ptr)
-@ stub D3DXIntersectSubset(ptr long ptr ptr ptr ptr ptr ptr ptr ptr ptr)
+@ stdcall D3DXIntersectSubset(ptr long ptr ptr ptr ptr ptr ptr ptr ptr ptr)
 @ stdcall D3DXIntersectTri(ptr ptr ptr ptr ptr ptr ptr ptr)
 @ stdcall D3DXLoadMeshFromXA(str long ptr ptr ptr ptr ptr ptr)
 @ stdcall D3DXLoadMeshFromXInMemory(ptr long long ptr ptr ptr ptr ptr ptr)
